functests/6_kubelet_testing/kubelet.go

```diff
@@ -0,0 +1,115 @@
+package __kubelet
```

Comment:
please add it under update tests, just create a new file under https://github.com/openshift-kni/performance-addon-operators/tree/master/functests/2_performance_update or you can just add test cases under the https://github.com/openshift-kni/performance-addon-operators/blob/master/functests/2_performance_update/updating_profile.go, no need to create additional directories

Reply:
okay sure will do.